tests/integration/upgrades/test_small_deployment_upgrades.py

```diff
@@ -276,7 +276,7 @@ async def test_upgrade_rollback_from_local(
         await wait_until(
             ops_test,
             apps=[app],
-            apps_statuses=["blocked"],
+            apps_statuses=["active"],
```

Comment:
This `blocked` allows an early escape from the `wait_until`, before the rollback is completed. Juju will fail the refresh command then if there are any units still executing.